server/service/apple_mdm.go

```diff
@@ -2161,46 +2162,25 @@ func (svc *MDMAppleCheckinAndCommandService) TokenUpdate(r *mdm.Request, m *mdm.
 			return err
 		}
 		if info.InstalledFromDEP {
-			svc.logger.Log("info", "running post-enroll commands in newly enrolled DEP device", "host_uuid", r.ID)
-			if err := svc.installEnrollmentPackages(r, m, info.TeamID); err != nil {
-				return fmt.Errorf("installing post-enrollment packages: %w", err)
-			}
+			svc.logger.Log("info", "queueing post-enroll task for newly enrolled DEP device", "host_uuid", r.ID)
 
-			if ref, ok := r.Params["enroll_reference"]; ok {
-				svc.logger.Log("info", "got an enroll_reference", "host_uuid", r.ID, "ref", ref)
-				appCfg, err := svc.ds.AppConfig(r.Context)
-				if err != nil {
-					return fmt.Errorf("getting app config: %w", err)
-				}
+			// TODO(mna): double-check/validate that r.ID == m.Enrollment.UDID == host.UUID
```

Comment:
I'm fairly sure this is correct (as we were already using both `r.ID` and `m.Enrollment.UDID` as host UUID in a few places) but I wanted to double-check before merging ^

Reply:
you're right, those are all the same thing.